assets/findify-filter.css

```diff
@@ -333,8 +333,35 @@ button {
   justify-content: space-between;
   padding: 0 12px 0 10px;
 }
+.price-range-content{
+  display:flex;
+  gap:5px;
+}
+
+.findify-price-range-input-container{
+  display:flex;
+  align-items: center;
+  padding: 0 8px;
+  border: 1px solid #000;
+  background: #fff;
+}
 .price-range-input{
   outline: none;
   box-shadow: none;
   max-width:50px;
+  display: block;
+  width: 100%;
+  margin: 0;
+  -webkit-box-sizing: border-box;
+  box-sizing: border-box;
+  height: 30px;
+  line-height: 30px;
+  padding-left: 8px;
+  background: transparent;
+  color: #111;
+  font-family: Montserrat,sans-serif;
+  font-size: 13px;
+  border: none;
+  outline: none;
+  z-index: 1;
```

Comment:
What is all this extra forced styling?
We are developing something reusable and flexible.
Use as less CSS as possible.

Reply:
Removed couple of rules here.